_i18n/es/resources/user-guides/prove-payment.md

```diff
@@ -1,86 +1,78 @@
 
 > get_tx_key TXID
 
-Alice would plug in her actual transaction ID instead of this TXID placeholder. All being well, the one time transaction key
-will be displayed.
+Alicia pondría su actual ID de transacción en lugar del marcador TXID. Si todo va bien, la llave de un solo uso será mostrada.
 
-Note that this will only work if monero-wallet-cli is set to save transaction keys. To double check:
+Tenga en cuenta que esto sólo funcionará si monero-wallet-cli está configurado para guardar llaves de transacción. Para revisar doblemente:
 
 > set
 
-If it's set to 0, set it to 1:
+Si set es 0, cámbialo a 1:
 
 > set store-tx-info 1
 
 #### GUI
 
-Alice can open her monero-wallet-gui and go to the history page to see her transaction detail:
+Alicia puede abrir monero-wallet-gui e ir al historial para ver los detalles de su transacción:
 
-![History](png/prove-payment/history.png)
+![Historial](png/prove-payment/history.png)
 
-Here, she can copy the transaction ID and Bob's address by clicking on each of them.
-Then she can click on `P` to get a payment proof (transaction key):
+Aquí, ella puede copiar el ID de transacción y la dirección de Bob haciendo clic en cada uno. Entonces ella puede hacer clic en `P` para obtener una prueba de pago (llave de transacción):
 
-![Payment proof](png/prove-payment/payment-proof.png)
+![Prueba de pago](png/prove-payment/payment-proof.png)
 
 
 ---
 
-Alice can now send Charlie the transaction key along with transaction ID and Bob's address.
+Alicia ahora puede enviarle a Charlie la llave de transacción junto al ID de transacción y la dirección de Bob.
 
-Note: if several transactions were made, this needs repeating for each such transaction.
+Nota: si varias transacciones fueron hechas, esto debe repetirse para cada transacción.
 
-### Check payments
+### Revisar pagos
 
-Charlie now received those three pieces of information, and wants to check Alice is telling the truth: on an up to date
-blockchain,
+Charlie ahora recibió esas tres piezas de información, y quiere revisar si Alicia está diciendo la verdad: en una blockchain actualizada,
 
 #### CLI
 
-Charlie types in monero-wallet-cli:
+Charlie escribe en monero-wallet-cli:
 
 > check_tx_key TXID TXKEY ADDRESS
 
-The information supplied by Alice plugs neatly instead of the placeholders. monero-wallet-cli will use the transaction
-key to decode the transaction, and display how much this particular transaction sent to this address. Obviously,
-Charlie will want to double check with Bob the address is really his - same as with Bitcoin.
+La información provista por Alicia se coloca pulcramente en lugar de los marcadores. monero-wallet-cli usará la llave de transacción para decodificar la transacción, y mostrar cuánto envió en particular la transacción a la dirección. Obviamente, Charlie querrá revisar doblemente con Bob que la dirección es realmente de él - al igual que con Bitcoin.
```

Comment:
Maybe working with some words to give a better translation. I also suggest removing the word "particular", it's redundant, and "revisar doblemente" is something not used commonly in Spanish, it even sounds bad.  

"La información provista por Alicia **encaja a la perfección**/ **encaja perfectammente** en lugar de los marcadores..., y mostrar cuánto envió la transacción a la dirección...Charlie querrá **volver a revisar**/ **nuevamente validar** con Bob..."

Reply:
I was not thinking very well when translating double check.

And about the "plugs neatly", it can be that way, but the sentence is talking about replacing the tags (TXID, TXKEY) for Alice's information.

Reply:
It's ok, sometimes these things happen. It has happened to me several times and I have to stop working and do something different, it helps :).

Yes, it says that when "you replace" the information it will "plug neatly". We could work out which terms to use, imo, the word "pulcramente" is not used very commonly and is a really formal word, in the Spanish dictionary (where I live is not really common, not sure about other regions; but those are my thoughts).

Reply:
It can be _limpiamente_ or _de manera limpia_ to not be too formal hehe.